app/assets/javascripts/pageflow/chart/editor/views/configuration_editor.js

```diff
@@ -13,6 +13,7 @@ pageflow.ConfigurationEditorView.register('chart', {
         required: true
       });
       this.view(pageflow.chart.DatawrapperAdView);
+      this.input('full_width', pageflow.CheckBoxInputView);
```

Comment:
The translation for this field displayed inside the editor, probably comes from the embedded video page type at the moment. Please add `config/locales/new/full_width.de.yml` and `full_width.en.yml` with the following translation keys:

```
en:
  pageflow:
    chart:
      page_attributes:
        full_width:
          inline_help: This text is displayed in the inline help displayed via a small "?" next to the field 
          label: This is the actual label text
```

After adding yml files you need to restart the development server. After that changes should take effect in the editor right away.

We should explain the structure of these translations in the ["Creating page types" guide](https://github.com/codevise/pageflow/blob/master/doc/creating_page_types.md#the-configuration-editor-schema).